Controller/TreeController.php

```diff
@@ -84,4 +102,48 @@ public function treeAction(Request $request)
             //'delete_in_overlay' => $deleteInOverlay ? $deleteInOverlay : false,
         ));
     }
+
+    /**
+     * Reorder $moved (child of $parent) before or after $target.
+     *
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function reorderAction(Request $request)
+    {
+        $parentPath = $request->get('parent');
+        $dropedAtPath = $request->get('dropped');
+        $targetPath = $request->get('target');
+        $position = $request->get('position');
+
+        if (null === $parentPath || null === $dropedAtPath || null === $targetPath) {
+            return new JsonResponse(['Parameters parent, dropped and target has to be set to reorder.'], Response::HTTP_BAD_REQUEST);
+        }
+
+        $before = 'before' == $position;
+        $parentNode = $this->session->getNode($parentPath);
```

Comment:
How is set the repository name please ? 
In the [resource controller](https://github.com/symfony-cmf/resource-rest-bundle/blob/master/src/Controller/ResourceController.php#L79) there is 
```
            $repository = $this->registry->get($repositoryName);

            $fullPath = method_exists($repository, 'resolvePath') ? $repository->resolvePath($path) : $path;
```